.circleci/config.yml

```diff
@@ -120,9 +120,12 @@ jobs:
       - image: datadog/docker-library:ansible_debian_2_10
     steps:
       - checkout
-      # TODO: upgrade ansible-lint to latest (will require updating the image for this task)
-      - run: pip install ansible-lint==6.17
-      - run: ansible-lint -v --exclude=.circleci/config.yml .
+      - run: pip install ansible-lint==6.17 galaxy-importer
+      - run: mkdir ansible_collections
+      - run: cd ansible_collections && ansible-galaxy collection init datadog.dd && mkdir -p datadog/dd/roles/agent/
+      - run: for file in $(find . -type f -and \( -path '*defaults*' -or -path '*tasks*' -or -path '*templates*' -or -path '*handlers*' -or -name '.gitkeep' -or -path '*meta*' -or -name 'README.md' \)); do echo "Create ansible_collections/datadog/dd/roles/agent/${file:2}" && mkdir -p $(dirname "ansible_collections/datadog/dd/roles/agent/${file:2}") && cp -r "$file" "ansible_collections/datadog/dd/roles/agent/${file:2}"; done
```

Comment:
This is not easy to read. What about putting this in a dedicated file and source it here? It would maybe help if we have further modifications, wdyt?

Reply:
I can give it it's dedicated file sure looks better this way for sure